pkgs/tools/security/dirbuster/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, stdenvNoCC, fetchurl, makeWrapper, copyDesktopItems, makeDesktopItem, binutils, jre, jdk }:
+
+let
+  name = "dirbuster";
+  version = "1.0-RC1";
+
+  binaryDist = stdenvNoCC.mkDerivation rec {
+    pname = "${name}-bin";
+    inherit version;
+
+    src = fetchurl {
+      url = "mirror://sourceforge/dirbuster/DirBuster%20(jar%20%2B%20lists)/${version}/DirBuster-${version}.tar.bz2";
+      sha256 = "1hgbamzlz83qxcl15nic2jr8542dgq0ghsvrqypc4bhsv7a2v0aj";
+    };
+
+    installPhase = ''
+      # Install dependency binaries
+      install -Dt $out/share/java lib/*
+
+      # Install wordlists
+      install -Dt $out/share/${name} *.{txt,jbrofuzz}
+    '';
+  };
+in
+stdenvNoCC.mkDerivation rec {
```

Comment:
I think we should use the `finalAttrs` pattern here.

The `finalAttrs` pattern will let you remove the `rec` keyword ([see its implementation in Nix](https://github.com/NixOS/nix/blob/2d1d81114d72ace89ce08cd3bc93f4eb27a2975d/src/libexpr/nixexpr.cc#L121)).

Why is this pattern preferred to `rec` ?

Let's take this simple code example:

```nix
mkDerivation rec {
   foo = 1;
   bar = foo + 1;
}
```

and then `.overrideAttrs(old: { foo = 2; })`, you'll get `{ foo = 2; bar = 2; }` while with `finalAttrs` pattern, it would work correctly because it's a real fixed point.

Let me share a couple of useful links regarding the `finalAttrs` pattern:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Feel free to reach out if you need some assistance.